### ISSUE TITLE
Inconsistent header and AccountHeader status bar padding

### ISSUE THREAD

**Issue by vytautasm** [mikepenz/MaterialDrawer#895]
Hey, I really love this library but I was having an issue that I solved in a weird way and I feel I should report this to help figure out why this might be happening.

Long story short: I add a custom header using `Drawer.setHeader()` after initialising the drawer. Full code:
       ` drawer = new DrawerBuilder()
            .withActivity(this)
            .withSelectedItem(-1)
            .build();`

A little after that I do:

`View header = inflater.inflate(R.layout.drawer_header_logged_in, null);
drawer.setHeader(header);`

The result is that on API 19+ I get weird padding that is the size of the status bar:
![2015-12-25 22 08 21](https://cloud.githubusercontent.com/assets/4701700/12004501/a2ac6976-ab66-11e5-96b0-8d7a48066400.png)

I didn't understand what was happening so I downloaded the sample app and went line by line (since the sample app displays the header correctly under the statusbar).

After a few tries I understood that `DrawerBuilder.withAccountHeader()` automatically fixes the problem. So now I changed my code to:

        drawer = new DrawerBuilder()
            .withActivity(this)
            .withSelectedItem(-1)
            .withAccountHeader(new AccountHeaderBuilder().withActivity(this).build())
            .build();

Basically I added a stub account header and now when I set my real header all looks good:

![2015-12-25 22 08 52](https://cloud.githubusercontent.com/assets/4701700/12004506/f8909e0c-ab66-11e5-9208-c7b6a2593436.png)

**Comment by mikepenz** [comment]
@vytautasm first thanks for your detailed issue report. 

I know that it seems that the behavior is may inconsistent, but it works as designed. If you use the `AccountHeader` everything is managed for you, and everything will be set up as it should look like, as of the **Material Design Guidelines**. 

If you now use the normal Header, it will be shown as normal `DrawerItem` on top. So it will also do not move below the `StatusBar` as this is not like `DrawerItems` should be displayed.
